bin/main.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+'Hello World!'
```

Comment:
Please, make the file executable by adding `#!/usr/bin/env ruby ` on the top line. Take some time to read about it [here](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/).